### ISSUE TITLE
[13] Add rel="nofollow" attribute to all external links

### ISSUE THREAD

**Issue by zainfathoni** [kawalcovid19/wargabantuwarga.com#44]
Since we don't have such granular control over Google Docs' links, we have to wait for #31 to be completed before proceeding with this improvement.

Hence this is currently blocked by #31.

**Comment by resir014** [comment]
Now that we have `htmr` installed, we should be able to perform these transform just by detecting any external links from the transform object.

```ts
const htmrTransform: HtmrOptions['transform'] = {
  // ...
  a: (node: JSX.IntrinsicElements['a']) => {
    const { href, children } = node

    if (href) {
      if (href.substr(0, 4) === 'http') {
        return (
          <Anchor href={href} target="_blank" rel="noopener noreferrer">
            {children}
          </Anchor>
        )
      }

      return (
        <Link href={href}>
          <a>{children}</a>
        </Link>
      )
    }

    return <a href={href}>{children}</a>
  }
}
```

**Comment by zainfathoni** [comment]
Wow, great! I didn't know that. 😅

**Comment by mazipan** [comment]
Also on the current homepage, the link contain:

```
https://www.google.com/url?q=
```

**Comment by zainfathoni** [comment]
Since we have swapped the home page with the content from Netlify CMS in #31, that Google URL prefix should have gone.

**Comment by hendraaagil** [comment]
I think this issue is already fixed 😅

**Comment by mazipan** [comment]
Thank you @hendraaagil

**Comment by mazipan** [comment]
@all-contributors please add @hendraaagil for code

**Comment by zainfathoni** [comment]
> I think this issue is already fixed 😅

This issue was not automatically closed because you mentioned this issue in the PR incorrectly. 😅

I just fixed it accordingly at #513, next time, please follow that convention to ensure that GitHub closes the related issues automatically. 😉

**Comment by hendraaagil** [comment]
Oh, I see. Sorry for my mistake 😆
